src/commands/analyze/index.ts

```diff
@@ -499,104 +512,106 @@ export default class Analyze extends Command {
       throw new Error(`Could not get receipt for ${transaction.hash}`)
     }
 
-    if (receipt.status === 1) {
-      let operatorJobHash: string | undefined
-      let operatorJobPayload: string | undefined
-      let args: any[] | undefined
-      let rawData: RawData | undefined
-
-      switch (this.environment) {
-        case Environment.localhost:
-          operatorJobHash = decodeCrossChainMessageSentEvent(receipt, this.networkMonitor.operatorAddress)
-          if (operatorJobHash !== undefined) {
-            args = decodeLzEvent(receipt, this.networkMonitor.lzEndpointAddress[network])
-            if (args !== undefined) {
-              operatorJobPayload = args[2] as string
-            }
-          }
+    if (receipt.status !== 1) {
+      throw new Error(`Transaction reverted ${transaction.hash}`)
+    }
 
-          break
-        default:
-          operatorJobHash = decodeCrossChainMessageSentEvent(receipt, this.networkMonitor.operatorAddress)
-          if (operatorJobHash !== undefined) {
-            operatorJobPayload = decodeLzPacketEvent(receipt, this.networkMonitor.messagingModuleAddress)
+    let operatorJobHash: string | undefined
+    let operatorJobPayload: string | undefined
+    let args: any[] | undefined
+    let rawData: RawData | undefined
+
+    switch (this.environment) {
```

Comment:
@BravoNatalie I would make this switch case a function. Maybe called `getJobHashAndPayload` ?